precompiles/batch/Cargo.toml

```diff
@@ -0,0 +1,57 @@
+[package]
+name = "pallet-evm-precompile-batch"
+description = "A Precompile to batch multiple calls."
+version = "0.1.0"
+authors = ["StakeTechnologies", "PureStake"]
+edition.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[dependencies]
+log = { workspace = true }
+num_enum = { workspace = true }
+paste = { workspace = true }
+slices = { workspace = true }
+
+# Moonbeam
+precompile-utils = { workspace = true }
+
+# Substrate
+frame-support = { workspace = true }
+frame-system = { workspace = true }
+parity-scale-codec = { workspace = true, features = ["max-encoded-len"] }
+sp-core = { workspace = true }
+sp-io = { workspace = true }
+sp-std = { workspace = true }
+
+# Frontier
+evm = { workspace = true, features = ["with-codec"] }
+fp-evm = { workspace = true }
+pallet-evm = { workspace = true, features = ["forbid-evm-reentrancy"] }
```

Comment:
I think `features = ["forbid-evm-reentrancy"]` is not needed as it's already specified in workspace `Cargo.toml`.